packages/rx_bloc_cli/README.md

```diff
@@ -19,19 +19,20 @@ $ dart pub global activate rx_bloc_cli
-|  `--enable-change-language` |                   `true`                    |                     Enables changing of the language for the project                      |
-|  `--enable-patrol` |                   `false`                   |               Enables patrol package for integration tests for the project                |
-|  `--realtime-communication`  |                   `none`                    |           Enables realtime communication facilities for SSE. Values: none, sse.           |
+| parameter                         |                 defaults to                 |                                        description                                        |
+|-----------------------------------|:-------------------------------------------:|:-----------------------------------------------------------------------------------------:|
+| `--project-name`                  | Name of directory where the project resides |  The project name for this new Flutter project. This must be a valid dart package name.   |
+| `--organisation`                  |                `com.example`                |                                   The organisation name                                   |
+| `--enable-analytics`              |                   `false`                   |                        Enables Firebase analytics for the project                         |
+| `--enable-feature-counter`        |                   `false`                   |                          Enables Counter feature for the project                          |
+| `--enable-feature-deeplinks`      |                   `false`                   |                     Enables Deep Links showcase flow for the project                      |
+| `--enable-feature-widget-toolkit` |                   `false`                   |              Enables widget_toolkit package showcase feature for the project              |
+| `--enable-social-logins`          |                   `false`                   | Integrate social login with Apple, Google and Facebook functionality into our application |
+| `--enable-change-language`        |                   `true`                    |                     Enables changing of the language for the project                      |
+| `--enable-patrol`                 |                   `false`                   |               Enables patrol package for integration tests for the project                |
+| `--realtime-communication`        |                   `none`                    |           Enables realtime communication facilities for SSE. Values: none, sse.           |
 | `--enable-dev-menu`               |                   `true`                    |                Enables dev menu to easily access proxy debugging services.                |
+| `--enable-otp`                    |                   `false`                   |          Enables OTP feature that can help with building sms/pin code workflows.          |
```

Comment:
I think here in the `defaults to` column , it should be `true`, because of the : packages/rx_bloc_cli/bin/compile.sh --enable-otp=true \

Reply:
Nope, in `packages/rx_bloc_cli/lib/src/commands/create_command.dart` it defaults to `false` so i think we refer to that one

Reply:
Ok, then I am wrong.